### ISSUE TITLE
Golden mage hand, ticking entity when they use bonemeal

### ISSUE THREAD

**Issue by simplyskellix** [Skylortrexler/MageHand#16]
[crash-2023-03-01_18.12.56-server (1).txt](https://github.com/skylorbeck/MageHand/files/10945913/crash-2023-03-01_18.12.56-server.1.txt)

edit; we do not have create, however it is a large modpack for the server.

**Comment by skylorbeck** [comment]
Do you have any mods that modify bonemeal?

**Comment by simplyskellix** [comment]
> 

I don't think so, the only thing I can think of is hex casting, you can grow crops with a spall, and we can also right-click crops with a hoe and harvest wheat.
